climada/hazard/tc_tracks_synth.py

```diff
@@ -76,6 +77,10 @@ def calc_random_walk(tracks,
             Default: configuration file
         decay (bool, optional): compute land decay in probabilistic tracks.
             Default: True
+
+
+    Returns:
+        List of xarray.Dataset instances representing cyclones
```

Comment:
Return None (remove the `Returns: ...` part of the docstring and make note in the body of the docstring that the input object is modified in place).